website/blog/2024-11-27-test-smarter-part-2.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Test smarter: Where should tests go in your pipeline?"
```

Comment:
One final question: Should this be 
```suggestion
title: "Test smarter not harder: Where should tests go in your pipeline?"
```
to have it be part of the TSNH cinematic universe?